Sources/StreamChat/StateLayer/Chat.swift

```diff
@@ -223,7 +223,7 @@ public class Chat {
     ///
     /// - Throws: An error while communicating with the Stream API.
     /// - Returns: An array of channel members for the pagination request.
-    @discardableResult public func loadMoreMembers(limit: Int? = nil) async throws -> [ChatChannelMember] {
+    @discardableResult public func loadMoreMembers(limit: Int = 30) async throws -> [ChatChannelMember] {
```

Comment:
how did you come up with the values? Did you take into account the backend defaults?

Reply:
These are just copy-pasted values from our `Int` extension. 

I put up this PR but I am actually not convinced anymore if it is a good idea.

Reply:
I agree, it's not. Better to pass nil and let the backend return its default.